### ISSUE TITLE
Can't run the installation script on High Sierra

### ISSUE THREAD

**Issue by codeams** [wakatime/xcode-wakatime#26]
Updated to Xcode 9, reinstalled and it was working fine.

After upgrading OS to high sierra and running

```
curl -fsSL https://raw.githubusercontent.com/wakatime/xcode-wakatime/master/install.sh | sh`
```

I'm getting:

```
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution Run\ Script build/WakaTime.build/Release/WakaTime.build/Script-E65F85E918D187D30082F26A.sh
(1 failure)
```

**Comment by HiroshiHorie** [comment]
Got the exact same error here.

**Comment by codeams** [comment]
Xcode usage is still being recorded even when I can no longer run the installation script.

This may be because I had xcode-wakatime successfully installed previous to the upgrade.

**Comment by alanhamlett** [comment]
The plugin's post-install script was failing because of a unicode filename in the wakatime-cli zip file being extracted. Fixed with bf54abf62b59b5bba3e30c93423c9c992c24bb32.
